src/main/groovy/edu/oregonstate/mist/invencycle/mapper/bikemapper.groovy

```diff
@@ -0,0 +1,50 @@
+package edu.oregonstate.mist.invencycle.mapper
+
+import edu.oregonstate.mist.invencycle.core.Bike
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+import java.sql.ResultSet
+import java.sql.SQLException
+
+public class bikemapper implements ResultSetMapper<Bike> {
```

Comment:
The naming of this class is incorrect. The convention in java is: CamelCase.java